audio/speech/getting-started/get_started_with_chirp_2_sdk.ipynb

```diff
@@ -0,0 +1,1331 @@
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "IOZNYPrfeW49"
+      },
+      "outputs": [],
+      "source": [
+        "audio_requests = (cloud_speech.StreamingRecognizeRequest(audio=s) for s in stream)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "oPbf5rNFecI_"
+      },
+      "source": [
+        "#### Define streaming recognition configuration\n",
+        "\n",
+        "Next, you define the streaming recognition configuration which allows you to set the model to use, language code of the audio and more."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "32Wz990perAo"
+      },
+      "outputs": [],
+      "source": [
+        "streaming_config = cloud_speech.StreamingRecognitionConfig(\n",
+        "    config=cloud_speech.RecognitionConfig(\n",
+        "        language_codes=[\"en-US\"],\n",
+        "        model=\"chirp_2\",\n",
+        "        features=cloud_speech.RecognitionFeatures(\n",
+        "            enable_automatic_punctuation=True,\n",
+        "        ),\n",
+        "        auto_decoding_config=cloud_speech.AutoDetectDecodingConfig(),\n",
+        "    )\n",
+        ")"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "zVRyTqhWe2gf"
+      },
+      "source": [
+        "#### Define the streaming request configuration\n",
+        "\n",
+        "Then, you use the streaming configuration to define the streaming request. In this case, you don't need to define a recognizer because you pass the model you want to use, language code and other settings in the streaming configuration above."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "t5qiUJ48e9i5"
+      },
+      "outputs": [],
+      "source": [
+        "stream_request = cloud_speech.StreamingRecognizeRequest(\n",
+        "    recognizer=f\"projects/{PROJECT_ID}/locations/{LOCATION}/recognizers/_\",\n",
+        "    streaming_config=streaming_config,\n",
+        ")"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "h1d508ScfD9I"
+      },
+      "source": [
+        "#### Run the streaming recognition request\n",
+        "\n",
+        "Finally, you are able to run the streaming recognition request."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "QCq-iROpfl9t"
+      },
+      "outputs": [],
+      "source": [
+        "def requests(config: cloud_speech.RecognitionConfig, s: list) -> list:\n",
+        "    yield config\n",
+        "    yield from s\n",
+        "\n",
+        "\n",
+        "response = client.streaming_recognize(requests=requests(stream_request, audio_requests))"
```

Comment:
Should the call to `requests()` include `stream_request` or `streaming_config`?

You might need to switch the parameter typing around.

Reply:
I fix a typo in the code.